src/System.Xml.XmlSerializer/src/netcore50/project.json

```diff
@@ -4,6 +4,7 @@
       "dependencies": {
         "Microsoft.NETCore.Targets": "1.0.0",
         "System.Collections": "4.0.10",
+        "System.Collections.NonGeneric": "4.3.0-beta-devapi-24503-01",
```

Comment:
Why are you referencing the latest version of this package instead of the stable version for the netcore50 build?